podcasts/Sharing/AVAsset+Crop.swift

```diff
@@ -0,0 +1,53 @@
+import AVFoundation
+
+extension AVAsset {
+    enum CropError: Error {
+        case failedVideoTrackCreation
+        case failedExportSession
+    }
+
+    func crop(to size: CGSize) async throws -> URL {
+        let composition = AVMutableComposition()
+        guard let compositionTrack = composition.addMutableTrack(withMediaType: .video, preferredTrackID: kCMPersistentTrackID_Invalid) else {
+            throw CropError.failedVideoTrackCreation
+        }
+
+        let assetTrack = try await loadTracks(withMediaType: .video)[0]
```

Comment:
Nit: maybe it will be more crash proof to have something like:
```
guard let assetTrack = try await loadTracks(withMediaType: .video).first else {
  throw CropError.noMediaAvailable
}
```